internal/inventory/asset.go

```diff
@@ -46,30 +48,50 @@ type AssetType string
 const (
 	TypeAcl                 AssetType = "acl"
 	TypeCloudAccount        AssetType = "cloud-account"
+	TypeDisk                AssetType = "disk"
 	TypeEventSource         AssetType = "event-source"
 	TypeFirewall            AssetType = "firewall"
 	TypeGateway             AssetType = "gateway"
 	TypeInterface           AssetType = "interface"
 	TypeLoadBalancer        AssetType = "load-balancer"
+	TypeNoSQLDatabase       AssetType = "nosql-database"
 	TypeNotificationService AssetType = "notification-service"
 	TypeObjectStorage       AssetType = "object-storage"
 	TypePeering             AssetType = "peering"
 	TypePolicy              AssetType = "policy"
+	TypeRegistry            AssetType = "registry"
 	TypeRelationalDatabase  AssetType = "relational"
+	TypeResourceGroup       AssetType = "resource-group"
 	TypeRole                AssetType = "role"
+	TypeScalability         AssetType = "scalability"
 	TypeServerless          AssetType = "serverless"
+	TypeSnapshot            AssetType = "snapshot"
+	TypeStorage             AssetType = "storage"
 	TypeSubnet              AssetType = "subnet"
 	TypeUser                AssetType = "user"
 	TypeVirtualMachine      AssetType = "virtual-machine"
 	TypeVirtualNetwork      AssetType = "virtual-network"
+	TypeWebApplication      AssetType = "web-application"
 )
 
 // AssetSubType is used to build the document index. Use only numbers, letters and dashes (-)
 type AssetSubType string
 
 const (
-	SubTypeAzureTenant              AssetSubType = "azure-tenant"
+	SubTypeAzureAppService          AssetSubType = "azure-app-service"
```

Comment:
hardly an issue, but there is some discrepancy between cloud providers subtypes. note that azure subtypes include `"azure-*` while AWS do not (for example: `ec2-instance`)

Reply:
All done to match [the classification](https://github.com/elastic/cloudbeat/blob/main/internal/inventory/ASSETS.md) provided by Tin.